services/payments/verifier.go

```diff
@@ -0,0 +1,41 @@
+package payments
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/brave-intl/bat-go/libs/httpsignature"
+)
+
+// the list of payment verifiers, mapping to individuals in payments-ops
+var validVerifiers = map[string]bool{
```

Comment:
Within the docs of this PR can we define the onboarding and offboarding process for this list? As a part of offboarding we should make sure the IT offboarding document gets this process updated so that when an operator leaves the company we make sure to get a PR in to disable their keys to authorize transactions.